### ISSUE TITLE
Fitting Window not scrolling on ship view

### ISSUE THREAD

**Issue by pincy** [pyfa-org/Pyfa#1843]
## Bug Report
In the *Fittings* tab, after scrolling down and selecting a ship from the bottom of the list, the window keeps in the scrolled down position.
I cannot see the actual fittings on this ship, nor can i scroll in the fittings tab, until i hit the new fit button, at which the window scrolls up, but i allways have to delete this additional fit.

### Expected behavior:
Fitting window scrolling up, when i selected the ship, or being able to scroll up.

### Detailed steps to reproduce:

* start pyfa
* switch to fittings tab
* select ship type
* cannot scroll in empty fittings window

### Release or development git branch? Please note the release version or commit hash:
Pulled from git, checkout v2.7.0

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Gentoo, python-3.6.5

`pip list`
```
alabaster (0.7.9)
alembic (0.9.7)
appdirs (1.4.3)
asn1crypto (0.22.0)
attrs (18.2.0)
Babel (2.5.3)
backports (1.0)
backports-abc (0.5)
bcrypt (3.1.3)
beautifulsoup4 (4.5.1)
bleach (2.1.3)
blinker (1.4)
bsddb3 (6.2.4)
cached-property (1.5.1)
certifi (2018.10.15)
cffi (1.11.4)
chardet (3.0.4)
chrome-gnome-shell (0.0.0)
click (6.7)
constantly (15.1.0)
cryptography (2.2.2)
cupshelpers (1.0)
cycler (0.10.0)
Cython (0.25.2)
decorator (4.2.1)
defusedxml (0.5.0)
diskcache (3.0.6)
Django (1.11.7)
django-cors-headers (2.1.0)
django-wkhtmltopdf (3.2.0)
djangorestframework (3.8.2)
djangorestframework-jwt (1.11.0)
djangorestframework-xml (1.3.0)
djangorestframework-yaml (1.0.3)
docker (2.7.0)
docker-compose (1.19.0)
docker-pycreds (0.3.0)
dockerpty (0.4.1)
docopt (0.6.2)
docutils (0.13.1)
entrypoints (0.2.3)
extras (1.0.0)
fixtures (3.0.0)
flaggie (0.2.1)
Flask (1.0.2)
Flask-Babel (0.11.2)
Flask-BabelEx (0.9.3)
Flask-Gravatar (0.5.0)
Flask-HTMLmin (1.3.2)
Flask-Login (0.3.2)
Flask-Mail (0.9.1)
Flask-Migrate (2.1.1)
Flask-Paranoid (0.2.0)
Flask-Principal (0.4.0)
Flask-Security (3.0.0)
Flask-SQLAlchemy (2.3.2)
Flask-WTF (0.14.2)
funcsigs (1.0.2)
gdbus-codegen (2.56.4)
gemato (14.0)
gentoolkit (0.4.2.post1)
gevent (1.2.1)
greenlet (0.4.12)
html5lib (1.0.1)
htmlmin (0.1.12)
idna (2.7)
imagesize (0.7.1)
incremental (16.10.1)
iotop (0.6)
ipykernel (4.6.1)
ipyparallel (6.0.2)
ipython (5.4.1)
ipython-genutils (0.2.0)
iso-639 (0.4.5)
iso3166 (0.8)
isodate (0.6.0)
itsdangerous (0.24)
java-config (2.2.0)
jedi (0.10.2)
Jinja2 (2.10)
jsonschema (2.6.0)
jupyter-client (5.1.0)
jupyter-core (4.4.0)
kiwisolver (1.0.1)
layman (2.4.2)
linecache2 (1.0.0)
livestreamer (1.12.2)
lockfile (0.11.0)
Logbook (1.1.0)
lxml (4.2.5)
lz4 (0.8.2)
Mako (1.0.6)
markdown2 (2.3.5)
MarkupSafe (0.23)
matplotlib (2.2.2)
meson (0.48.2)
mimeparse (1.6.0)
mistune (0.8.3)
mps-youtube (0.2.8)
nbconvert (5.2.1)
nbformat (4.4.0)
nbxmpp (0.6.1)
ndg-httpsclient (0.4.0)
netifaces (0.10.6)
notebook (5.0.0)
numpy (1.14.5)
olefile (0.44)
packaging (17.1)
pafy (0.5.3.1)
pandocfilters (1.4.2)
passlib (1.7.1)
path.py (8.1.2)
pathlib2 (2.3.0)
Paver (1.2.4)
paypalrestsdk (1.13.1)
pbr (4.2.0)
pexpect (4.2.1)
pickleshare (0.7.4)
piexif (1.0.13)
Pillow (5.0.0)
pip (9.0.1)
pluggy (0.5.2)
ply (3.9)
portage (2.3.51)
prompt-toolkit (1.0.15)
psycopg2 (2.7.4)
ptyprocess (0.5.1)
py (1.5.4)
py-sg (0.11)
pyasn1 (0.2.3)
pyasn1-modules (0.0.8)
pyblake2 (1.1.2)
pycairo (1.17.0)
pycodestyle (2.3.1)
pycparser (2.14)
pycrypto (2.6.1)
pycryptodome (3.6.0)
pycryptodomex (3.6.6)
pycups (1.9.73)
pycurl (7.43.0)
Pygments (2.2.0)
pygobject (3.28.3)
pyGPG (0.2)
PyJWT (1.6.4)
pymssql (2.1.3)
pyOpenSSL (17.5.0)
pyparsing (2.2.0)
Pypubsub (4.0.0)
pyrsistent (0.14.2)
pysmi (0.1.3)
pysnmp (4.4.4)
PySocks (1.6.7)
pytest (3.4.2)
pytest-runner (2.6.2)
python-dateutil (2.7.2)
python-editor (1.0.3)
python-gnupg (0.4.1)
python-mimeparse (1.6.0)
python-subunit (1.2.0)
pytz (2018.5)
pyxattr (0.6.0)
pyxdg (0.25)
PyYAML (3.13)
pyzmq (16.0.2)
rdflib (4.2.2)
redis (2.10.5)
regex (2017.4.5)
rencode (1.0.5)
reportlab (3.4.0)
requests (2.20.0)
requests-toolbelt (0.8.0)
rfc3987 (1.3.7)
roman (3.0)
scandir (1.7)
scour (0.36)
scrypt (0.8.0)
service-identity (16.0.0)
setuptools (36.7.2)
setuptools-scm (1.15.6)
simplegeneric (0.8.1)
simplejson (3.13.2)
sip (4.19.13)
six (1.11.0)
snowballstemmer (1.2.1)
speaklater (1.3)
Sphinx (1.7.5)
sphinx-bootstrap-theme (0.6.5)
sphinx-rtd-theme (0.1.9)
sphinxcontrib-websupport (1.1.0)
SQLAlchemy (1.2.7)
sqlparse (0.2.4)
ssl-fetch (0.4)
stevedore (1.28.0)
streamlink (0.11.0)
strict-rfc3339 (0.6)
terminado (0.5)
testpath (0.1)
testtools (2.3.0)
texttable (0.9.1)
tornado (4.5.3)
traceback2 (1.4.0)
traitlets (4.3.2)
Twisted (16.6.0)
unittest2 (1.1.0)
urllib3 (1.24)
vcversioner (2.16.0.0)
versioneer (0.16)
virtualenv (15.1.0)
virtualenv-clone (0.2.6)
virtualenvwrapper (4.8.2)
wcwidth (0.1.7)
webcolors (1.5)
webencodings (0.5.1)
websocket-client (0.48.0)
Werkzeug (0.14.1)
Whoosh (2.7.4)
WTForms (2.1)
wxPython (4.0.3)
youtube-dl (2019.1.17)
zeep (3.1.0)
zope.interface (4.4.3)
```

**Comment by blitzmann** [comment]
Hrm, this isn't a problem for me on Windows. And we have enough linux users where I would say one would have mentioned this. I wonder if it's specific to your environment at all...

I notice that you're on wxPython 4.0.3. While I doubt thats the issue, I don't know how well tested it is.

Can you take a few screenshots / video that better demonstrates this issue?

If I can't reproduce, we'll have to try blind fixed (one of which might be simply setting the scroll of the window during navigational events, thought I'm not show how possible that is)

**Comment by pincy** [comment]
Hi,

so i made a short screen recording. [Here](https://gfycat.com/FinishedLonelyGreathornedowl)
The behaviour you can see there is just the same, with other menu levels.
Like, scroll down to Rattlesnake under Battleship, click on Rattlesnake, don't see any of the 3 saved fittings.

**Comment by DarkFenX** [comment]
I have this issue as well btw. But for now for me impact has been negligible (i alsost always use search to find all these leshaks/barghests/rattlesnakes), so didn't bother to report/fix. If you have any ideas on how to fix it - poke me, i can easily test it.

**Comment by blitzmann** [comment]
I did peak real quick to see if setting the scroll is possible, and it is indeed. Can look at the logic used for the "jump to ship browser" context item when jumps to the proper panel and then scrolls to the item in question. See https://github.com/pyfa-org/Pyfa/blob/9d75dea31aefa204f7146bbae890fefdd646ea93/gui/builtinShipBrowser/pfListPane.py#L71

Granted, that's scrolling a child into view -we may be able to just shortcut it and do something like `self.Scroll(0, 0)`. Will have to determine where exactly to put this statement in the GUI code to be most beneficial (assuming after each stage is loaded, but there's some funky things that can happen that we might need to account for). Will try to tackle this sometime this week :)

**Comment by blitzmann** [comment]
Should be fixed with 65dff8d :D pretty easy fix, simply set the scroll every time the panel is refreshed

**Comment by pincy** [comment]
Testing here, resolved my issues.
Thank you
